### PR TITLE
landing page v-1

### PR DESCRIPTION
**Pull request by symaticvisuals**
### Description

Landing Page version-1

### CODE REVIEW

**Review by Abhishek-kumar09 — COMMENTED** [review]
Why you have changed the previous home page?
Create a new home page without deleting the previous one.

**Comment by symaticvisuals** [comment]
I haven't deleted it it's still there I just have commented the call for it because as per design the landing page will be first then the hoke page will come. Am I right?

**Comment by Abhishek-kumar09** [comment]
The svg is not there on homepage.
> I haven't deleted it it's still there I just have commented the call 

Don't comment the call, instead make a different route to show that page

**Comment by symaticvisuals** [comment]
> The svg is not there on homepage.
> 
> > I haven't deleted it it's still there I just have commented the call
> 
> Don't comment the call, instead make a different route to show that page

Sure

**Comment by Abhishek-kumar09** [comment]
@symaticvisuals Any update on this?

**Comment by symaticvisuals** [comment]
Yes, I have worked on that but can you tell me where should I create the route for that, because If I create it in the routes files then am not getting the navbar

**Review by symaticvisuals — COMMENTED** [review]
@Abhishek-kumar09 please help me with some responsiveness of the page I have created the design and it's looking fine for the desktop but not for the mobile

**Comment by Abhishek-kumar09** [comment]
@symaticvisuals Where is the previous landing page, I can't find that. It was also important.

**Comment by symaticvisuals** [comment]
@Abhishek-kumar09 I haven't removed it just changed the file name. I will fix that today for sure to avoid confusion

**Review by symaticvisuals — COMMENTED** [review]
@Abhishek-kumar09 the landing page is done you can check it out , it is just not responsible for the tablets , else I have tested some mobiles and desktops.
The route has been added at /landing

File: src/views/pages/HomeView/LandingPage.js [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+  Button: {
+    textTransform: 'capitalize'
+    // padding :
+  },
+  container: {
+    paddingTop: '10vh',
+    margin: '0 5px',
+    [theme.breakpoints.down('sm')]: {
+      margin: '0',
+      paddingTop: '2vh',
```
Comment (zeborg) [inline src/views/pages/HomeView/LandingPage.js:62]: Try setting it to `6vh`, seems to fix the issue with tablet viewport here.

Comment (zeborg) [reply src/views/pages/HomeView/LandingPage.js:62]: By _it_ I mean `paddingTop`.

Comment (symaticvisuals) [reply src/views/pages/HomeView/LandingPage.js:62]: Oh ok , I will do it tomorrow. 
Thank you

Comment (zeborg) [reply src/views/pages/HomeView/LandingPage.js:62]: Wait, I think there's an even bigger issue at hand here! The `Footer.svg` blob seems to be broken for most of the viewports. I'll try working on a solution for this from my end as well. Screenshots below for reference:

image
image
image
image
image

Comment (symaticvisuals) [reply src/views/pages/HomeView/LandingPage.js:62]: Yes am also facing this issue , but this is only showing on chrome., on firefox its not happening

**Comment by symaticvisuals** [comment]
@Abhishek-kumar09 should I work on some other issue now?

**Comment by Abhishek-kumar09** [comment]
Yes you can if you want to.
Just let me know the problems with the current PR

**Comment by symaticvisuals** [comment]
The footer image is broken for breakpoints and I have to add the tablet compatibility @Abhishek-kumar09 I think rest is fine

**Comment by symaticvisuals** [comment]
@Abhishek-kumar09 will this Pr will merge? Or I need to make more changes?

**Comment by Abhishek-kumar09** [comment]
This PR can't be merged untill all the required chages are made

**Comment by symaticvisuals** [comment]
> This PR can't be merged untill all the required chages are made

@Abhishek-kumar09 do let me know changes I will do it

**Comment by Abhishek-kumar09** [comment]
https://github.com/codeforcauseorg/causefolio/pull/59#discussion_r647806314
I think responsiveness is the biggest issue here. Try to make it responsive

**Comment by symaticvisuals** [comment]
> https://github.com/codeforcauseorg/causefolio/pull/59#discussion_r647806314
> I think responsiveness is the biggest issue here. Try to make it responsive

@Abhishek-kumar09 actually it's responsive just the footer svg is breaking on size change with black background how can I fix that

**Review by Abhishek-kumar09 — APPROVED** [review]
The landing page is looking perfect on all screens.
Very Nice work @symaticvisuals

**Pull request merged**